daemon/server/local_scoot.go

```diff
@@ -0,0 +1,56 @@
+package server
```

Comment:
I'd put RunManager somewhere in scoot/runner (and rename it) unless there's going to be more daemon specific stuff added.